src/components/ToolTip.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module">
+  import HelpStrings from "../data/help";
+
+  export const getHelpMessage = (key) => {
```

Comment:
I don't think we need the indirection of this function (and the one below it) -- I'd just grab the message / links inside the component and pass them as properties here.